haystack/components/evaluators/context_relevance.py

```diff
@@ -111,8 +120,8 @@ def __init__(
                     "questions": "What is the capital of Italy?", "contexts": ["Rome is the capital of Italy."],
                 },
                 "outputs": {
-                    "statements": ["Rome is the capital of Italy."],
-                    "statement_scores": [1],
+                    "relevant_statements": ["Rome is the capital of Italy."],
+                    "score": 1,
```

Comment:
```suggestion
```

The `_DEFAULT_EXAMPLES` do not contain `score`, so I assume it is not needed.
Please also update the docstring to reflect the structure of the example.

Reply:
this is correct - this evaluator outputs a binary score, 1 or 0 if the a statement is considered relevant - why do you want you remove it?

Reply:
I think you confused prompt/instructions to the LLM with the component's output

Reply:
ah no, wait you are right, sorry - my confusion